test/Feature/AudienceJwtSecretSpec.hs

```diff
@@ -0,0 +1,21 @@
+module Feature.AudienceJwtSecretSpec where
+
+-- {{{ Imports
+import Test.Hspec
+import Test.Hspec.Wai
+import Network.HTTP.Types
+
+import SpecHelper
+import Network.Wai (Application)
+
+import Protolude hiding (get)
+-- }}}
+
+spec :: SpecWith Application
+spec = describe "server started with binary JWT secret" $
```

Comment:
This description threw me off for a second because its emphasis seems to be about the binary secret rather than the audience claim.

Reply:
I'll revise to be more clear what this is testing, and add decoded versions of the JWT in comments so the difference is clear to the reader.

Reply:
I also extended the unit test to show that if jwt-aud is configured, and the JWT presented does not have an `aud` claim, that the validation will still succeed. Seems like we are great for backward compatibility.